src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -78,7 +78,12 @@ public CompilationTaskEventArgs(CompilationTaskEventType type, string parentTask
         /// If LoadAssembly is not null, it will be used to load the dlls that are search for classes defining rewrite steps.
         /// </summary>
         public static Func<string, Assembly> LoadAssembly { get; set; }
-
+        /// <summary>
+        /// Sorts the given list of step according to their relative priority give by getPriority. 
+        /// Throws the corresponding exception if getPriority is null. 

```

Comment:
```suggestion
        /// Throws the corresponding exception if <paramref name="getPriority" /> is <c>null</c>. 

```

Reply:
BTW, I've found it useful on IQ# to start using `#nullable enable` to help document what reference types support `null` and which cannot be `null`.